Ome/JavaScript/Assignment 3/JavaScriptAssignment3_Q3.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous" />
+    <title>JavaScript - Check Number Is Even Or Odd</title>
+    <style>
+        .container-box {
+            margin: auto;
+            margin-top: 20px;
+            width: 50%;
+        }
+    </style>
+</head>
+
+<body class="container">
+    <div class="container-box">
+        <h1>Check Given Number Is Even Or Odd</h1>
+
+        <label class="form-label">Please Enter Number:</label><br />
+        <div class="col-3">
+            <input type="number" id="putNumber" class="form-control" /><br />
+        </div>
+        <button type="submit" onclick="chkNum()" class="btn btn-primary">
+            Check
+        </button>
+
+        <p>
+            <br />
+            <strong id="show"></strong>
+        </p>
+    </div>
+
+    <script>
+        function chkNum() {
+            var num = parseInt(document.getElementById("putNumber").value);
+
+            if (num % 2 == 0)
+                document.getElementById("show").innerHTML = num + " is prime number";
```

Comment:
it is program for checking even and odd not for prime

Reply:
Thank you poonam for highlighting this. I made changes in the file please check.